expat/configure.ac

```diff
@@ -94,6 +94,9 @@ AC_LANG_POP([C++])
 AS_IF([test "$GCC" = yes],
   [AX_APPEND_LINK_FLAGS([-fno-strict-aliasing],[LDFLAGS])])
 
+AS_IF([test "$GCC" = yes],
```

Comment:
That looks more like non-clang and not like a check of the linker. What is the idea?

Reply:
Updated LD check for clang linker in new patchset